Patch 1 (#6)
aragon/help: user-guide/introduction/prerequisites.md

		<li>Note that most mobile Web3 browsers have a built-in Ethereum provider and wallet functionality.</li>
	</ul>
	</li>

> 	<li><strong>If you are using Aragon on mainnet</strong>, you will need to set your Ethereum provider to use the Main Ethereum Network ("mainnet"). Mainnet is default, but you can double check by looking your address bar in the browser https://mainnet.aragon.org/ (You will then need to load your wallet with enough ETH to pay for gas when you deploy your organization and make transactions within the organization. ETH can be acquired from a <a href="https://docs.ethhub.io/using-ethereum/how-to-buy-ether/" target="_blank">cryptocurrency exchange that lists ETH</a>. As a rough estimate of how much ETH you should start with, multiply 20,000,000 times the <a href="https://ethgasstation.info/" target="_blank">current “standard” gas price</a> times 0.000000001 to find out how much ETH you need. For example, at a current standard gas price of 10 gwei, you should start with 0.2 ETH.</li>

Reviewer: ```suggestion
	<li><strong>If you are using Aragon on mainnet</strong>, first go to <a href="https://mainnet.aragon.org" target="_blank">https://mainnet.aragon.org</a>. Then set your Ethereum provider to use the Main Ethereum Network ("mainnet") and load your wallet with enough ether ("ETH") to pay for gas when you deploy your organization and make transactions within the organization. ETH can be acquired from a <a href="https://docs.ethhub.io/using-ethereum/how-to-buy-ether/" target="_blank">cryptocurrency exchange that lists ETH</a>. As a rough estimate of how much ETH you should start with, multiply 20,000,000 times the <a href="https://ethgasstation.info/" target="_blank">current “standard” gas price</a> times 0.000000001 to find out how much ETH you need. For example, at a current standard gas price of 10 gwei, you should start with 0.2 ETH.</li>
```

> 	<li><strong>If you are using Aragon on testnet</strong>, you will need to set your Ethereum provider to use the Rinkeby Test Network ("Rinkeby"). To switch to testnet, just navigate to Aragon on Rinkeby by going to the address https://rinkeby.aragon.org/#/ . Be sure to also change your wallet app to the Rinkby test network and load your wallet with test-ETH. This test-ETH is used to pay for gas when you deploy your organization and make transactions within the organization. You can try the <a href="https://faucet.rinkeby.io/" target="_blank">Rinkeby faucet</a> or post your Rinkeby testnet address in the <a href="https://aragon.chat/faucet" target="_blank">#faucet channel on Aragon Chat</a> to receive test-ETH for free. Please wait patiently for an Aragon community member to see your message if you post your faucet request in the Aragon Chat.</li>

Reviewer: ```suggestion
	<li><strong>If you are using Aragon on testnet</strong>, first go to <a href="https://rinkeby.aragon.org" target="_blank">https://rinkeby.aragon.org</a>. Then set your Ethereum provider to use the Rinkeby Test Network ("Rinkeby") and load your wallet with test-ETH. This test-ETH is used to pay for gas when you deploy your organization and make transactions within the organization. You can try the <a href="https://faucet.rinkeby.io/" target="_blank">Rinkeby faucet</a> or post your Rinkeby testnet address in the <a href="https://aragon.chat/faucet" target="_blank">#faucet channel on Aragon Chat</a> to receive test-ETH for free. Please wait patiently for an Aragon community member to see your message if you post your faucet request in the Aragon Chat.</li>
```
